tests/api.c

```diff
@@ -13178,11 +13189,19 @@ static int test_wc_AesCbcEncryptDecrypt (void)
         if (cbcD == BAD_FUNC_ARG) {
             cbcD = wc_AesCbcDecrypt(&aes, dec, enc, AES_BLOCK_SIZE * 2 - 1);
         }
+#ifdef WOLFSSL_AES_CBC_ALIGNMENT_CHECKS
```

Comment:
I'd be fine with having the logic `cbcD == BAD_ALIGN_E || cbcD == BAD_FUNC_ARG`, but this is good, since its explicit.

Reply:
Yeah and in fact I've come to think it's important to check for the old return value on the default build -- thinking of ABI and breaking incumbent app code.